### ISSUE TITLE
Possible bug in method template?

### ISSUE THREAD

**Issue by sbusch** [mtennoe/swagger-typescript-codegen#62]
I don't know Mustache well enough, but I think I found a bug in [templates/method.mustache](https://github.com/mtennoe/swagger-typescript-codegen/blob/master/templates/method.mustache):

`isBodyParameter` is accessed **outside** of the `{{#parameters}}`loop at both methods in that template:

https://github.com/mtennoe/swagger-typescript-codegen/blob/777e5ca3cdef5c0d161e351a1c148b7d6c2e1478/templates/method.mustache#L45

https://github.com/mtennoe/swagger-typescript-codegen/blob/777e5ca3cdef5c0d161e351a1c148b7d6c2e1478/templates/method.mustache#L158

**Comment by mtennoe** [comment]
Yep, you're right, this looks wrong! Is this affecting you currently?

**Comment by sbusch** [comment]
No, just found it while making other changes

**Comment by mtennoe** [comment]
Great! Let's fix this as soon as someone has the time. Ofc, if someone reports that they hit this we will jump on it sooner.
Thanks for reporting!
